### PR TITLE
Replaced DeveloperErrors with Check for CylinderOutlineGeometry.js

### PR DESCRIPTION
**Pull request by rms13**
Worked on previously discussed issue #4794 , for file Source/Core/CylinderOutlineGeometry.js

### CODE REVIEW

**Comment by pjcozzi** [comment]
Awesome, thanks @rms13!

@omh1280 can you please review this?

File: Source/Core/CylinderOutlineGeometry.js [hunk 1]
```diff
@@ -72,18 +72,10 @@ define([
...
-        if (!defined(bottomRadius)) {
-            throw new DeveloperError('options.bottomRadius must be defined.');
-        }
-        if (slices < 3) {
-            throw new DeveloperError('options.slices must be greater than or equal to 3.');
-        }
+        Check.typeOf.number('options.positions', length);
+        Check.typeOf.number('options.topRadius', topRadius);
+        Check.typeOf.number('options.bottomRadius', bottomRadius);
+        Check.typeOf.number.greaterThanOrEquals('options.sclices', slices, 3);
```
Comment (omh1280) [inline Source/Core/CylinderOutlineGeometry.js]: Minor spelling mistake, `sclices`

Comment (rms13) [inline Source/Core/CylinderOutlineGeometry.js]: Oh! Sorry about that. I'll update it.

Comment (pjcozzi) [inline Source/Core/CylinderOutlineGeometry.js]: @rms13 please bump this PR when you push the commit so we know to merge it.

Comment (rms13) [inline Source/Core/CylinderOutlineGeometry.js]: Done!

**Comment by pjcozzi** [comment]
Congrats on your first pull request!

Thanks for the review, @omh1280.

**Pull request merged**